platforms/core-configuration/model-core/src/test/groovy/org/gradle/internal/properties/annotations/DefaultTypeMetadataStoreTest.groovy

```diff
@@ -95,7 +95,7 @@ class DefaultTypeMetadataStoreTest extends Specification implements ValidationMe
         cacheFactory
     )
     def propertyTypeResolver = new DefaultPropertyTypeResolver()
-    def metadataStore = new DefaultTypeMetadataStore([], services.getAll(PropertyAnnotationHandler), [Classpath, CompileClasspath], typeAnnotationMetadataStore, propertyTypeResolver, cacheFactory)
+    def metadataStore = new DefaultTypeMetadataStore([], services.getAll(PropertyAnnotationHandler), [Classpath, CompileClasspath], typeAnnotationMetadataStore, propertyTypeResolver, cacheFactory, MissingPropertyAnnotationHandler.DO_NOTHING)
```

Comment:
❔ As all these tests used to have the `MISSING_INPUT_OUTPUT_HANDLER` behavior, maybe they should still add that handler, even if it won't be called?

Reply:
I guess my thinking was that the purpose of these tests is not to test what happens when a property is missing an annotation, so if they behave differently without the handler, then that's a problem.  In other words, fewer moving parts makes the test more targeted at the functionality it is actually intending to test.  I don't feel strongly about this, though, so if you disagree, I'm willing to be convinced.

Reply:
Nah, I don't feel strongly about this either way, just pointing out the change in case it mattered.